README.md

```diff
@@ -150,49 +173,85 @@ However, some trees are constructed iteratively from unsorted data, causing the
 import { StandardMerkleTree } from "@openzeppelin/merkle-tree";
 ```
 
-### `StandardMerkleTree.of`
+#### `StandardMerkleTree.of`
 
 ```typescript
 const tree = StandardMerkleTree.of([[alice, '100'], [bob, '200']], ['address', 'uint'], options);
 ```
 
 Creates a standard merkle tree out of an array of the elements in the tree, along with their types for ABI encoding. For documentation on the syntax of the types, including how to encode structs, refer to the documentation for Ethers.js's [`AbiCoder`](https://docs.ethers.org/v5/api/utils/abi/coder/#AbiCoder-encode).
 
-> **Note**
-> Consider reading the array of elements from a CSV file for easy interoperability with spreadsheets or other data processing pipelines.
+#### `StandardMerkleTree.load`
 
-> **Note**
-> By default, leaves are sorted according to their hash. This is done so that multiproof generated by the library can more easily be verified onchain. This can be disabled using the optional third argument. See the [Leaf ordering](#leaf-ordering) section for more details.
```

Comment:
I think these two notes should be kept.

Reply:
Yeah, good call. They don't match where they were before so I readded them at the top of the API & Examples section.